Compile error without libtls

```
constraint-disabled.c:46:1: error: conflicting types for ‘priv_constraint_msg’
 priv_constraint_msg(u_int32_t id, u_int8_t *data, size_t len,
 ^~~~~~~~~~~~~~~~~~~
In file included from constraint-disabled.c:19:0:
ntpd.h:380:7: note: previous declaration of ‘priv_constraint_msg’ was here
 void  priv_constraint_msg(u_int32_t, u_int8_t *, size_t, int, char **);
       ^~~~~~~~~~~~~~~~~~~
Makefile:570: recipe for target 'ntpd-constraint-disabled.o' failed
```
constraint-disabled.c needs update.
see https://github.com/openntpd-portable/openntpd-openbsd/commit/c8ed30f0cf3eef0a6aa9c36b34a4236458ca7fa2

--- a/src/constraint-disabled.c
+++ b/src/constraint-disabled.c
@@ -33,6 +33,11 @@ constraint_query(struct constraint *cstr)
 }
 
 void
+priv_constraint_child(const char *pw_dir, uid_t pw_uid, gid_t pw_gid)
+{
+}
+
+void
 priv_constraint_check_child(pid_t pid, int status)
 {
 }
@@ -43,8 +48,8 @@ priv_constraint_kill(u_int32_t id)
 }
 
 void
-priv_constraint_msg(u_int32_t id, u_int8_t *data, size_t len,
-    const char *pw_dir, uid_t pw_uid, gid_t pw_gid)
+priv_constraint_msg(u_int32_t id, u_int8_t *data, size_t len, int argc,
+    char **argv)
 {
 }
 

